ckanext/bcgov/logic/get.py

```diff
@@ -0,0 +1,139 @@
+# encoding: utf-8
+
+'''API functions for searching for and getting data from CKAN.'''
+
+import logging
+
+import sqlalchemy
+from paste.deploy.converters import asbool
+
+import ckan.logic as logic
+import ckan.lib.navl.dictization_functions
+import ckan.model as model
+
+from ckan.logic.action.get import ( _unpick_search )
+
+log = logging.getLogger('ckanext.bcgov.logic.bcgov.override')
+
+# Define some shortcuts
+# Ensure they are module-private so that they don't get loaded as available
+# actions in the action API.
+_validate = ckan.lib.navl.dictization_functions.validate
+ValidationError = logic.ValidationError
+
+_or_ = sqlalchemy.or_
+
+# Copied from ckan 2.7.7 ckan.logic.action.get
+# Modified for the puposes of wrapping the asbool in a try:catch
+def _group_or_org_list(context, data_dict, is_org=False):
+    model = context['model']
```

Comment:
Oops, i should have been a bit more clear, python doc string are underneath python functions, modules, etc.

eg.
```
def _group_or_org_list(...):
    """
    Copied from ckan 2.7.2 ckan.logic.action.get

    Modified for the purposes of catching `asbool` value error 
    due to a bug in the function with missing argument value, eg. `None`

Reply:
updated.